doc/api/http.md

```diff
@@ -1446,6 +1467,17 @@ added: v12.7.0
 Is `true` if all data has been flushed to the underlying system, immediately
 before the [`'finish'`][] event is emitted.
 
+### response.writableEnded
```

Comment:
This section needs to be placed before `### response.writableFinished` alphabetically.